Meta/run.sh

```diff
@@ -183,6 +183,17 @@ else
     SERENITY_QEMU_DISPLAY_DEVICE="VGA,vgamem_mb=64 "
 fi
 
+# FIXME: Is there a better way of enabling NVME support?
+SERINITY_NVME_ENABLE=0
+if [ "$SERINITY_NVME_ENABLE" -eq 1 ]; then
+
+    SERENITY_NVME_ARGS="-drive file=${SERENITY_DISK_IMAGE},format=raw,index=0,media=disk,if=none,id=disk"
+    SERENITY_NVME_ARGS="$SERENITY_NVME_ARGS -device i82801b11-bridge,id=bridge4 -device sdhci-pci,bus=bridge4"
+    SERENITY_NVME_ARGS="$SERENITY_NVME_ARGS -device nvme,serial=deadbeef,drive=disk,bus=bridge4"
+else
+    SERENITY_NVME_ARGS="-drive file=${SERENITY_DISK_IMAGE},format=raw,index=0,media=disk"
```

Comment:
Rename to SERENITY_BOOT_DRIVE?

Reply:
This is definitely a better name than `SERENITY_NVME_ARGS`. Will use this in my next revision.